translations/monero-core_id.ts

```diff
@@ -1079,404 +1079,411 @@ The following information will be deleted
         <source>This sets which outputs are known to be spent, and thus not to be used as privacy placeholders in ring signatures. </source>
-        <translation type="unfinished"></translation>
+        <translation>Mengatur keluaran mana yang boleh dibelanjakan, dan dengan demikian tidak digunakan sebagai penampung privasi di tanda cincin.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="144"/>
         <source>You should only have to load a file when you want to refresh the list. Manual adding/removing is possible if needed.</source>
-        <translation type="unfinished"></translation>
+        <translation>Anda seharusnya hanya memuat file ketika Anda ingin me-refresh daftar. Penambahan/penghapusan manual dimungkinkan jika diperlukan.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="156"/>
-        <source>Please choose a file from which to load outputs to mark as spent</source>
-        <translation type="unfinished"></translation>
+        <source>Please choose a file to load blackballed outputs from</source>
+        <translation>Pilih file untuk memuat keluaran blackball</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="172"/>
         <source>Path to file</source>
-        <translation type="unfinished"></translation>
+        <translation>Alur ke file</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="174"/>
-        <source>Filename with outputs to mark as spent</source>
-        <translation type="unfinished"></translation>
+        <source>Filename with outputs to blackball</source>
+        <translation>Nama file dengan keluaran ke blackball</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="187"/>
         <source>Browse</source>
-        <translation type="unfinished"></translation>
+        <translation>Jelajahi</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="198"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>Memuat</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="214"/>
-	<source>Or manually mark a single output as spent/unspent:</source>
-        <translation type="unfinished"></translation>
+        <source>Or manually blackball/unblackball a single output:</source>
+        <translation>Atau secara manual mem-blackball/mengurungkan blackball suatu keluaran:</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="215"/>
         <source>Paste output amount</source>
-        <translation type="unfinished"></translation>
+        <translation>Tempelkan jumlah keluaran</translation>
+    </message>
```

Comment:
Double closing tag here. Remove this string please

Reply:
Done